faq.md: add proxy question (#752)
pingcap/docs-dm: en/faq.md

When a DM-master starts, DM records the etcd information in the current directory. If the directory changes after the DM-master restarts, DM cannot get access to the etcd information, and thus the restart fails.

To solve this issue, you are recommended to maintain DM clusters using TiUP. In the case that you need to deploy using binary files, you need to configure `data-dir` with absolute paths in the configuration file of the DM-master, or pay attention to the current directory where you run the command.

> ## Why DM-master cannot be connected when using dmctl to execute commands?

Reviewer: not sure if dmctl should be wrapped with backquotes

Author: In command-line-flags.md, dmctl is considered as a component just like DM-master and DM-worker. In addition, most of the dmctl in the document are not wrapped. So I think we don't have to wrap it.

When using dmctl execute commands, you might find the connection to DM master fails (even if you have specified the parameter value of `--master-addr` in the command), and the error message is like `RawCause: context deadline exceeded, Workaround: please check your network connection.`. But afer checking the network connection using commands like `telnet <master-addr>`, no exception is found.

In this case, you can check the environment variable `https_proxy` (note that it is **https**). If this variable is configured, dmctl automatically connects the host and port specified by `https_proxy`. If the host does not have a corresponding `proxy` forwarding service, the connection fails.

> To solve this issue, confirm whether `https_proxy` must be configured. If not, cancel the setting. Otherwise, add the environment variable setting `https_proxy="" ./dmctl --master-addr "x.x.x.x:8261"` before the oringial dmctl commands.

Reviewer 2: ```suggestion
To solve this issue, check whether `https_proxy` is mandatory. If not, cancel the setting. Otherwise, add the environment variable setting `https_proxy="" ./dmctl --master-addr "x.x.x.x:8261"` before the oringial dmctl commands.
```

> **Note:**
> 

> > The environment variables related to `proxy` include `http_proxy`, `https_proxy`, `no_proxy`, etc. If the connection to DM master still fails after you perform the above steps, check whether the configuration parameters of `http_proxy` and `no_proxy` made a difference.

Reviewer 2: ```suggestion
The environment variables related to `proxy` include `http_proxy`, `https_proxy`, and `no_proxy`. If the connection error persists after you perform the above steps, check whether the configuration parameters of `http_proxy` and `no_proxy` are correct.
```
